### ISSUE TITLE
Init fails out on Windows

### ISSUE THREAD

**Issue by achekulaev** [docksal/drupal7#5]
```
{ ~ }  » fin create-site                                                                                                    ~
One second! Checking for updates...
Please enter the name of your project (all one word, no spaces or slashes): demo

What would you like to install?
[1] Drupal 7
[2] Drupal 8
[3] Wordpress

: 1

Your site will be created at /home/alex/demo
Your site will run Drupal 7
The URL of your site will be http://demo.docksal

Do you wish to proceed? [y/n]: y
 Cloning repository...
Cloning into 'demo'...
remote: Counting objects: 2034, done.
remote: Total 2034 (delta 0), reused 0 (delta 0), pack-reused 2034
Receiving objects: 100% (2034/2034), 3.79 MiB | 1.99 MiB/s, done.
Resolving deltas: 100% (652/652), done.
Checking connectivity... done.
 Step 1  Initializing local project configuration...
Copying /home/alex/demo/docroot/sites//settings.local.php...
cp: cannot stat ‘/home/alex/demo/docroot/sites//example.settings.local.php’: No such file or directory

{ ~ }  » cd demo                                                                                                            ~

{ demo } master » ls                                                                                                   ~/demo
docroot  drush  README.md  tests

{ demo } master » fin init                                                                                             ~/demo
 Step 1  Initializing local project configuration...
Copying /home/alex/demo/docroot/sites//settings.local.php...
cp: cannot stat ‘/home/alex/demo/docroot/sites//example.settings.local.php’: No such file or directory
```

**Comment by achekulaev** [comment]
Same issue with https://github.com/docksal/drupal8

**Comment by achekulaev** [comment]
Resolved in fin 0.13
